README.md

```diff
@@ -41,7 +41,7 @@ When working with WSL, the host vscode instance connects to a vscode server on t
 
 - Install the `pokey.command-server` extension on the host vscode
 - Clone the extension in the WSL side.
-- If you're adding language support to `vscode-parse-tree`, you need to clone that as well, build it, and link it into the `vscode-server` extension folder: `ln -s ~/.vscode-server/extensions/vscode-parse-tree ~/git/vscode-pars-tree` for instance.
+- If you're adding language support to `vscode-parse-tree`, you need to clone that as well, build it, and link it into the `vscode-server` extension folder: `ln -s ~/your/code/vscode-parse-tree ~/.vscode-server/extensions/parse-tree` for instance.
```

Comment:
I get this wrong literally every time I use `ln -s`, glad to see I'm not the only one

Reply:
The way I remember it is that it's the same order as `mv` or `cp`: source to destination